### PR TITLE
[#23] Framework update and bug fixes related to it

### PR DESCRIPTION
**Pull request by mihaiiftode**
There are two of them because sendgrid nugget is closely linked to the way you use it.

### CODE REVIEW

**Comment by alexandru-calinoiu** [comment]
On a quick visual inspection looks like the testimonials part of the site is broken

capture

File: Agilefreaks/Controllers/ContactUsController.cs [hunk 1]
```diff
@@ -4,50 +4,46 @@
...
 namespace Agilefreaks.Controllers
 {
-    using System.Threading.Tasks;
-
-    using SendGrid;
-
     public class ContactUsController : Controller
     {
-        private const string ContactEmail = "office@agilefreaks.com";
+        private const string ContactEmail = "ofice@agilefreaks.com";
```
Comment (alexandru-calinoiu) [inline Agilefreaks/Controllers/ContactUsController.cs]: De ce l-ai facut cu `ofice`?

Comment (mihaiiftode) [inline Agilefreaks/Controllers/ContactUsController.cs]: misspell, nu am observat

**Comment by mihaiiftode** [comment]
Please review the changes again, as most of the commits have changed

**Pull request merged**